recipes-core/images/includes/licenses.inc

```diff
@@ -1,16 +0,0 @@
-
-COPY_LIC_MANIFEST = "1"
```

Comment:
Can you remove the `l` mixin in the [images/README](https://github.com/ni/meta-nilrt/blob/9a3f91edebf83a4598944cc3bb5871d14b1e235b/recipes-core/images/README#L8)?

Reply:
Done.